index.html

```diff
@@ -20,29 +20,67 @@
+                    <li><a href="#about-section" class="menu-item">About</a></li>
+                    <li><a href="#contact-section" class="menu-item">Contact</a></li>
                 </ul>
             </nav>
         </div>
     </header>
-    
+
+
     <!-- mobile-menu -->
     <div class="mobile-menu hidden">
         <div class="mobile-menu-container">
-            <div onclick="closeNav()" class="close-btn" id="close-nav">
+            <div onclick="closeView()" class="close-btn" id="close-nav">
                 <i class="fas fa-times"></i>
             </div>
-             <!-- Content of the mobile menu -->
+            <!-- Content of the mobile menu -->
             <ul class="mobile-menu-list">
-                <li class="mobile-menu-list-items" onclick="closeNav()"><a href="#works-section">Portfolio</a></li>
-                <li class="mobile-menu-list-items" onclick="closeNav()"><a href="#about-section">About</a></li>
-                <li class="mobile-menu-list-items" onclick="closeNav()"><a href="#contact-section">Contact</a></li>
+                <li class="mobile-menu-list-items" onclick="closeView()"><a href="#works-section">Portfolio</a></li>
+                <li class="mobile-menu-list-items" onclick="closeView()"><a href="#about-section">About</a></li>
+                <li class="mobile-menu-list-items" onclick="closeView()"><a href="#contact-section">Contact</a></li>
             </ul>
         </div>
     </div>
-<!-- mobile-menu END-->
+    <!-- mobile-menu END-->
+    <!-- Details view -->
+    <div class="detail-view hidden" id="detail-view">
+    <div class="detail-view-container">
+        <button onclick="closeView()" class="detail-close-btn" id="close-view">
+            <!-- <i class="fas fa-times desktop-close"></i> -->
+            </button>
+           
+        <div class="image-container"></div>
+        <h2 class="detail-title" id="detail-title"></h2>
+<div class="detail-tags">
+    <ul>
+        <li>Ruby on Rails</li>
+        <li>CSS</li>
+        <li>Javascript</li>
+    </ul>
+</div>
+<div class="detail-summary">
+
+<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+    standar dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+<p>
+<p>Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+    standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it
+    1960s.</p>
+</div>
+
+
+     <div class="detail-btn-container">
+<button class="detail-btn">See Live<a href="https://rileymanda.github.io/portfolio-website/"><img src="./assets/icons/live-icon.svg"
+            alt="Twitter icon"></a></button>
+<button class="detail-btn">See Source<a href="https://github.com/RileyManda/portfolio-website"><img src="./assets/icons/github-light.svg" alt="Twitter icon"></a></button>
+     </div>
+        
```

Comment:
- [ ] In this particular project, the section of the whole project along with the models should be created dynamically with Javascript. I can see that the base projects section was populated with the data from the `workcards` array good job! 👏🏻 , but the Modal popup data section is still not being created with Javascript.  Please use Javascript to dynamically render your Modal popup windows data as you did with the projects section.

- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `projectList` array. 😉 👍🏻 
 
 - (Note: the works section must be removed from HTML and generated with javascript )